addons/mail/static/src/core/common/message_reaction_list.js

```diff
@@ -0,0 +1,88 @@
+import { Component, useState } from "@odoo/owl";
+import { Dropdown } from "@web/core/dropdown/dropdown";
+import { useDropdownState } from "@web/core/dropdown/dropdown_hooks";
+
+import { _t } from "@web/core/l10n/translation";
+import { useService } from "@web/core/utils/hooks";
+
+export class MessageReactionList extends Component {
+    static template = "mail.messageReactionList";
+    static components = { Dropdown };
+    static props = ["message", "openReactionMenu", "reaction"];
+
+    setup() {
+        super.setup();
+        this.store = useState(useService("mail.store"));
+        this.ui = useService("ui");
+        this.hover = useHover(["reactionButton", "reactionList*"], () => {
+            this.preview.isOpen = this.hover.isHover;
+            if (this.hover.isHover) {
+                clearTimeout(this.showReactionListTimeout);
+                this.showReactionListTimeout = setTimeout(() => {
+                    this.state.showReactionList = true;
+                }, 25);
+            } else if (!this.hover.isHover) {
+                clearTimeout(this.showReactionListTimeout);
+                this.state.showReactionList = false;
+            }
+        });
+        this.preview = useDropdownState();
+        this.state = useState({ showReactionList: false, hightlight: false });
+    }
+
+    /** @param {import("model").MessageReactions} reaction */
+    getReactedPersonaNames(reaction) {
+        const personaNames = reaction.personas
+            .map(({ name, displayName }) => name || displayName)
+            .slice(0, 3);
+        return _t(personaNames.join(", "), ...personaNames);
+    }
+
+    /** @param {import("model").MessageReactions} reaction */
+    getReactionContentAndCount(reaction) {
+        const { count, content } = reaction;
+
+        switch (count) {
+            case 1:
+                return _t(" has reacted with %s", content);
+            case 2:
+            case 3:
+                return _t(" have reacted with %s", content);
+            case 4:
+                return _t(" and 1 other person have reacted with %s", content);
+            default:
+                return _t(" and %(count)s other persons have reacted with %(content)s", {
+                    count: count - 3,
+                    content,
+                });
+        }
+    }
+
+    hasSelfReacted(reaction) {
+        return this.store.self.in(reaction.personas);
+    }
+
+    onClickReaction(reaction) {
+        if (this.hasSelfReacted(reaction)) {
+            reaction.remove();
+        } else {
+            this.props.message.react(reaction.content);
+        }
+    }
+
+    onContextMenu(ev) {
+        if (this.ui.isSmall) {
+            ev.preventDefault();
+            this.props.openReactionMenu();
+        }
+    }
+
+    handleMouseEvent(ev) {
+        if (ev.type === "mouseenter") {
+            this.state.hightlight = true;
+        } else {
+            this.state.hightlight = false;
+        }
+    }
```

Comment:
As said before, this could be done fully in CSS, so I removed it.
But the value was very low so I just removed it.